docs/tips.md

```diff
@@ -180,7 +180,7 @@ To set a wallpaper with the "Open with" menu (<kbd>O</kbd> key by default), add
 ```toml
 # Linux: Hyprland + Hyprpaper
 [[opener.set-wallpaper]]
-run  = 'cp "$0" "$HOME/.config/hypr/wallpaper" && systemctl restart --user hyprpaper.service'
+run  = 'hyprctl hyprpaper reload ,"$1"'
```

Comment:
Is there any reason you're using `$1` (first selected file) instead of `$0` (hovered file)?

Reply:
nope, i just copied it from other command

$0 seems better